### PR TITLE
Disable tournament creation button while saving

### LINKED ISSUE #54: Quickly creating tournaments doesn't show
There might be a problem with can-connect.

If you enter a date in the New Tournament __Date__ field and hit "Create" twice quickly, only one tournament shows up despite making two requests.

Likely this is due to `.save()` being called twice on the same instance.  

This is probably fixed by using `{{isSaving}}` to disable the create button.

cursor_and_localhost_5000_dev_html__

### PR DESCRIPTION
**Pull request by akagomez**
Use `{{tournament.isSaving}}` to disable the submit button. 

Closes #54.

### CODE REVIEW

**Comment by justinbmeyer** [comment]
Probably an "inline" funcunit test against fixtures.  By inline, I mean you are adding this component to a local test area, and using funcunit to click the button twice rapidly.

**Comment by akagomez** [comment]
This PR won't be mergeable until syn does a release with this PR: https://github.com/bitovi/syn/pull/110.

I attempted to do it myself, but don't have the NPM privileges. Neither does @matthewp.

File: public/components/tournament/test.js [hunk 1]
```diff
@@ -1 +1,43 @@
...
+F.attach(QUnit);
+
+QUnit.module('Tournament List', {
+    beforeEach: function () {
+        localStorage.clear();
+    }
+});
+
+QUnit.test('Create button is disabled while posting data', function () {
+    var frag = can.stache('<tournament-list tournament="{tournament}"></tournament-list>')({
```
Comment (akagomez) [inline public/components/tournament/test.js]: {tournament}="tournament"

Comment (akagomez) [inline public/components/tournament/test.js]: Self close the component element.

File: public/components/tournament/test.js [hunk 2]
```diff
@@ -1 +1,43 @@
...
+        tournament: {
+            name: 'Ballderdash',
+            date: '01/21/1987'
+        }
+    });
+
+    $('#qunit-fixture').html(frag);
+
+    // Ensure fixture requests take some amount of time
+    can.fixture.delay = 250;
```
Comment (akagomez) [inline public/components/tournament/test.js]: Don't do this.

Comment (akagomez) [inline public/components/tournament/test.js]: Create fixture inline.

File: public/components/tournament/test.js [hunk 3]
```diff
@@ -1 +1,43 @@
...
+import F from 'funcunit';
+import './list';
+import fixture, { tournamentData } from 'bitballs/models/tournament/fixture'
+import './details/test';
+
+F.attach(QUnit);
+
+QUnit.module('Tournament List', {
+    beforeEach: function () {
+        localStorage.clear();
```
Comment (akagomez) [inline public/components/tournament/test.js]: Setup fixtures here.

File: public/components/tournament/test.js [hunk 4]
```diff
@@ -1 +1,59 @@
...
+
+    // Click the button multiple times and ensure it's disabled
+    // during requests
+    F('tournament-list .create-btn')
+        .visible('Create button is visible')
+        .attr('disabled', undefined, 'Create button is enabled')
+        .click()
+        .attr('disabled', 'disabled', 'Create button is disabled after click')
+        .click()
+        .wait(fixture.delay)
```
Comment (akagomez) [inline public/components/tournament/test.js]: It turns out `can-fixture` has a default `delay` of 200ms, so the waiting is still necessary. However, I did update this PR with the rest of your feedback.

**Comment by akagomez** [comment]
Once the tests pass I'll merge.

**Comment by akagomez** [comment]
Resolving the merge conflict. Will have to finish in the morning.

**Comment by akagomez** [comment]
Rebased. Tests should pass.

**Pull request merged**